Blockzilla/URLBar.swift

```diff
@@ -142,7 +147,7 @@ class URLBar: UIView {
         clearButton.setImage(#imageLiteral(resourceName: "icon_clear"), for: .normal)
         clearButton.addTarget(self, action: #selector(didPressClear), for: .touchUpInside)
 
-        urlText.font = UIConstants.fonts.urlText
+        urlText.font = UIConstants.fonts.urlText.withSize(16)
```

Comment:
We should probably update the urlText constant to have this size by default, unless it will change throughout runtime?